### ISSUE TITLE
Check posenet_to_kinect2d.ipynb Notebook

### ISSUE THREAD

**Issue by angelicagardner** [digitacs/4dv652-ml#8]
Please check this Notebook: [posenet_to_kinect2d.ipynb](https://github.com/digitacs/4dv652-ml/blob/DIS-118-ml_performance_experimentation/notebooks/assignment12/posenet_to_kinect2d.ipynb)

**Questions from me:**

1. What is missing from this setup that you need for the pipeline? And is there something that needs to be changed/adjusted?
2. I would like your help with the scaler, where am I supposed to reverse the scaling and how will that affect the setup?

Thanks.

**Comment by ArioAtlas** [comment]
we can write configurations like `optimizer = Adam(learning_rate=learning_rate)`, the available data type would be string or numbers (i.e. we cannot use classes). therefore, I guess we should use the factory design pattern here:
1. replace the mentioned line with `optimizer =  ADAM` (then use a factory to create Adam object)
2. I guess you should extract the early stopping function's parameters as well ( I mean these: monitor='val_loss',  verbose=1, patience=20, mode='min', restore_best_weights=True )
3. In order to experience different layer configurations, you should extract layers and put them in the configuration section as well
following is a recommended sample. for configuration file:
`layers = [
    { type: 'Dense', nodes:32, activation: 'relu' },
    { type: 'Dense', nodes:32, activation: 'sigmoid' },
    { type: 'Dense', nodes:3, activation: 'softmax' },
]`

you can remove the `output_layer` variable because is already included in the above configuration
4. for history please use the last assignment version, I've used a configuration for all metrics and plots and merge them in a single image

**Comment by ArioAtlas** [comment]
I will add a code block for reverse scaling later today (very soon, I promise :D)

**Comment by angelicagardner** [comment]
Many thanks for the feedback. 
I made some corrections: [posenet_to_kinect2d.ipynb](https://github.com/digitacs/4dv652-ml/blob/DIS-118-ml_performance_experimentation/notebooks/assignment12/posenet_to_kinect2d.ipynb)
However, I don't understand what you mean by adding layers into the configuration file. Could you please explain that a bit more?

Also, second notebook in case you want to check that out: [kinect2d_to_3d.ipynb](https://github.com/digitacs/4dv652-ml/blob/DIS-118-ml_performance_experimentation/notebooks/assignment12/kinect2d_to_3d.ipynb)
It has the same setup, so you don't necessarily have to go through that one unless you want to.

**Comment by ArioAtlas** [comment]
@angelicagardner 
I push a new commit to DIS-118
in the future, please start your commit description with your card prefix (DIS-118) and use card prefix (DIS-118) only for the branch name
